.eslintrc.js

```diff
@@ -5,7 +5,7 @@ module.exports = {
     "react-hooks",
     "@typescript-eslint",
     "styled-components-a11y",
-    "jest",
+    // "jest",
```

Comment:
This is just me trying to quiet things down. We should obviously re-enable this.